api/app/weather_models/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 # Key values on ModelRunGridSubsetPrediction.
 # Wind direction (wdir_tgl_10_b) is handled slightly differently, so not included here.
```

Comment:
Very minor, but I think this comment is no longer needed, unless `wdir_tgl_10_**b**` is something different

Reply:
Actually maybe this is why I'm not getting bias adjusted values for wind dir...